src/werkzeug/debug/shared/debugger.js

```diff
@@ -20,51 +42,44 @@ docReady(function() {
 });
 
 function initPinBox() {
-    $('.pin-prompt form').submit(function(evt) {
-        evt.preventDefault();
-        var pin = this.pin.value;
-        var btn = this.btn;
-        btn.disabled = true;
-        $.ajax({
-            dataType: 'json',
-            url: document.location.pathname,
-            data: {
-                __debugger__: 'yes',
-                cmd: 'pinauth',
-                pin: pin,
-                s: SECRET
-            },
-            success: function(data) {
-                btn.disabled = false;
-                if (data.auth) {
+    document.querySelector(".pin-prompt form")
+        .addEventListener("submit", function(event) {
```

Comment:
Can we use arrow syntax here to be consistent?

Reply:
oh my bad, just saw we're using `this` in the function so this should be good as is.